io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+        sha256: e4ef3b845f121882c7c0cf04f81a1cb8fd360c6f64b56b868de5546214f904de
+    config-opts:
+      - --with-guile-site=yes
+
+  - name: guile-gnome-platform
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/guile-gnome/guile-gnome-platform/guile-gnome-platform-2.16.5.tar.gz
+        sha256: 298d8c4f9b567bfe87beda18ed58d047c2e01b88c80895129de5466b921ccebe
+    buildsystem: autotools
+    build-options:
+      - CFLAGS: "-Wall"
+
+  - name: gemmi
+    sources:
+      - type: git
+        url: https://github.com/project-gemmi/gemmi.git
+        commit: a928991bc31983d99b3aeffc01794313b6638613
+    buildsystem: cmake
+
+  - name: Catch2
+    sources:
+      - type: archive
+        url: https://github.com/catchorg/Catch2/archive/refs/tags/v3.7.1.tar.gz
+        sha256: c991b247a1a0d7bb9c39aa35faf0fe9e19764213f28ffba3109388e62ee0269c
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - "-DCMAKE_CXX_STANDARD=17"
+      - "-DCMAKE_PREFIX_PATH=/app"
+
+  - name: eigen
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.3.9/eigen-3.3.9.tar.gz
+        sha256: 7985975b787340124786f092b3a07d594b2e9cd53bbfe5f3d9b1daee7d55f56f
+    build-commands:
+      - cmake -S . -B eigen-build -Dpkg_config_libdir=/app/lib/pkgconfig -DCMAKE_INSTALL_PREFIX=/app
+      - cmake --install eigen-build --prefix=/app
+      - install -D cmake/FindEigen3.cmake /app/share/cmake/Modules/FindEigen3.cmake
+
+  - name: maeparser
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/schrodinger/maeparser
+        tag: v1.3.1
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/maeparser
+      - cp -r * /app/tmp/maeparser
+
+  - name: coordgenlibs
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/schrodinger/coordgenlibs
+        tag: v3.0.2
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/coordgen
+      - cp -r * /app/tmp/coordgen
+
+  - name: RingDecomposerLib
+    sources:
+      - type: git
+        url: https://github.com/rareylab/RingDecomposerLib
+        tag: v1.1.3_rdkit
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/RingDecomposerLib
+      - cp -r * /app/tmp/RingDecomposerLib
+
+  - name: pubchem-align3d
+    sources:
+      - type: git
+        url: https://github.com/ncbi/pubchem-align3d
+        commit: daefab3dd0c90ca56da9d3d5e375fe4d651e6be3
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/pubchem-align3d
+      - cp -r * /app/tmp/pubchem-align3d
+
+  - name: rapidjson
+    sources:
+      - type: git
+        url: https://github.com/Tencent/rapidjson
+        commit: 858451e5b7d1c56cf8f6d58f88cf958351837e53
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/tmp/rapidjson-1.1.0
+      - cp -r * /app/tmp/rapidjson-1.1.0
```

Comment:
you don't install in `/app/tmp`. Same for the other. It's non standard.

Reply:
I need to download them before RDKit build.

When finishing this build process, they will be clean up. 
https://github.com/flathub/flathub/pull/5690/files#diff-55b9e84a877dbbef6cbca71f44f2c253217232100c0d81f840e1abf5346b24efR511

Reply:
no. see above.

Reply:
https://github.com/flathub/flathub/pull/5690?notification_referrer_id=NT_kwDOAUFjg7QxMjc2ODI2MzY2MzoyMTA2MjUzMQ#discussion_r1794837739